otherlibs/systhreads/thread.mli

```diff
@@ -184,3 +184,9 @@ val set_uncaught_exception_handler : (exn -> unit) -> unit
 
     If the newly set uncaught exception handler raise an exception,
     {!default_uncaught_exception_handler} will be called. *)
+
+val set_extended_uncaught_exception_handler: (exn -> Printexc.raw_backtrace -> unit) -> unit
```

Comment:
How about `set_uncaught_exception_handler_with_backtrace`?
It gives more detail in the function name than `extended`